### PR TITLE
LPS-169605

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-169605

### CODE REVIEW

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/messaging/CheckNotificationQueueEntryMessageListener.java [hunk 1]
```diff
@@ -15,84 +15,153 @@
...
-	protected void activate() {
-		Class<?> clazz = getClass();
-
-		String className = clazz.getName();
+	public CheckNotificationQueueEntryMessageListener(
+		long companyId, ConfigurationProvider configurationProvider,
+		SchedulerEngineHelper schedulerEngineHelper,
+		TriggerFactory triggerFactory,
+		NotificationQueueEntryLocalService notificationQueueEntryLocalService,
+		NotificationTypeServiceTracker notificationTypeServiceTracker) {
```
Comment (marcelabc) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/messaging/CheckNotificationQueueEntryMessageListener.java:55]: Could you sort these parameters please?

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/messaging/CheckNotificationQueueEntryMessageListener.java [hunk 2]
```diff
@@ -15,84 +15,153 @@
...
+
+		_configurationProvider = configurationProvider;
+		_schedulerEngineHelper = schedulerEngineHelper;
+		_triggerFactory = triggerFactory;
+		_notificationQueueEntryLocalService =
+			notificationQueueEntryLocalService;
+		_notificationTypeServiceTracker = notificationTypeServiceTracker;
```
Comment (marcelabc) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/messaging/CheckNotificationQueueEntryMessageListener.java:62]: Could you sort these variables please?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
Sent to: https://github.com/liferay-core-infra/liferay-portal/pull/2878

cc: @marcelabc @samuelromeiro

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**